### ISSUE TITLE
Null pointer to a converted model using Tensorflow Lite in Android Studio. "Byte buffer is not a valid flatbuffer model." Exception

### ISSUE THREAD

**Issue by mainouraldeen** [tensorflow/tensorflow#37173]
i made a model containing layers of type: Conv3D, MaxPool3D, Flatten, Dense and Batch normalization and converted it to tflite model as follows..

```bash
import tensorflow as tf

converter = tf.lite.TFLiteConverter.from_saved_model(saved_model_dir)
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS,
                                       tf.lite.OpsSet.SELECT_TF_OPS]
tflite_model = converter.convert()
```

The problem is when i load this converted model in Android Studio it always gives me illegal argument exception saying  **"Byte buffer is not a valid flatbuffer model"** & a null pointer to it

i also, specified this dependency in my build.gradle to add the necessary TF op support.

```bash
dependencies {
    implementation 'org.tensorflow:tensorflow-lite:0.0.0-nightly'
    implementation 'org.tensorflow:tensorflow-lite-select-tf-ops:0.0.0-nightly'
} 
```
Any help?

**Comment by gowthamkpr** [comment]
@mainouraldeen use tensorflow-nighty build gradle version 0.1.2
` implementation 'org.tensorflow:tensorflow-lite:0.1.2-nightly'`

For more information Please go through this i[ssue](https://stackoverflow.com/questions/56256478/converted-model-tflite-does-not-encode-a-valid-tensorflowlite-model-could-not-o/56269570).

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by ravikyram** [comment]
@mainouraldeen 

Any update on this issue please. Thanks!

**Comment by gowthamkpr** [comment]
Closing this issue as it has been inactive for 4 weeks. Please add additional comments for us to open this issue again. Thanks!
